pkg/services/contexthandler/contexthandler.go

```diff
@@ -70,6 +71,11 @@ func (h *ContextHandler) Middleware(c *macaron.Context) {
 		Logger:         log.New("context"),
 	}
 
+	traceID, exists := cw.ExtractTraceID(c.Req.Request.Context())
```

Comment:
Ah. I missed this. Perhaps we can delete https://github.com/grafana/grafana/blob/master/pkg/middleware/logger.go#L64